index.js

```diff
@@ -0,0 +1,45 @@
+const Assyst = require('./lib/Assyst.js').Assyst
+const config = require('./config.json')
+
+const client = new Assyst(config)
+
+console.log('Starting Assyst')
+
+console.log('Loading commands')
+
+client.loadCommands()
+
+client.bot.on('messageCreate', (msg) => {
```

Comment:
This could also totally be directly part of your client class, or even in another class. But it's fine here.